app/controllers/support_interface/application_forms_controller.rb

```diff
@@ -2,7 +2,9 @@ module SupportInterface
   class ApplicationFormsController < SupportInterfaceController
     def index
       @filter = SupportInterface::ApplicationsFilter.new(params:)
-      @application_forms = @filter.filter_records(ApplicationForm)
+      result = @filter.filter_records(ApplicationForm)
+      @pagy = result[:pagy]
+      @application_forms = result[:records]
```

Comment:
Why is this different compared to the usual `@pagy, @application_forms = pagy(applications_forms)`